### PR TITLE
merge leg conditions

### PR DESCRIPTION
**Pull request by tamnguyenit**
more information https://github.com/wego/curiosity/pull/798

**Related:** 
https://github.com/wego/makalu/pull/416
https://wegomushi.atlassian.net/browse/WEB-459

### CODE REVIEW

File: src/flight-search/Merger.js [hunk 1]
```diff
@@ -99,6 +100,17 @@ FlightSearchMerger.prototype = {
     });
   },
 
+  _mergeLegConditions: function(newLegConditions) {
+    if (!newLegConditions) return;
+    var self = this;
+    var legMap = this.__legMap;
+    Object.keys(newLegConditions).forEach(function(legId) {
+      if (legMap[legId]) {
+        legMap[legId]["conditionIds"] = newLegConditions[legId]
```
Comment (wilson-alcantara) [inline src/flight-search/Merger.js]: Can you append semi-colon ;

Comment (tamnguyenit) [reply src/flight-search/Merger.js]: done boss @wilson-alcantara

File: src/flight-search/Merger.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ FlightSearchMerger.prototype = {
 
     this._mergeStaticData(response);
     this._mergeLegs(response.legs);
+    this._mergeLegConditions(response.legConditionIds)
```
Comment (wilson-alcantara) [inline src/flight-search/Merger.js]: This one too, and run build then commit and all is well sir

Comment (tamnguyenit) [reply src/flight-search/Merger.js]: haha, ok boss

**Pull request merged**